skeleton/acp/main_module.php.tpl

```diff
@@ -19,7 +19,7 @@ class main_module
 
 	function main($id, $mode)
 	{
-		global $user, $template, $request, $config;
+		global $config, $request, $template, $user;
```

Comment:
Instead of using all those globals, we could just use ```global $phpbb_container``` and then get all needed services with its ```->get``` method.